packages/venia-concept/src/components/Navigation/categoryLeaf.css

```diff
@@ -6,6 +6,11 @@
     width: 100%;
 }
 
+.inactive {
+    position: fixed;
```

Comment:
I worry that this combination will cause an invisible click target which shadows other targets. Have you made sure that inactive elements are totally out of the viewport in all cases?

Reply:
Yeah, we should revisit this as part of the upcoming accessibility epic.